lib/_kv.ts

```diff
@@ -53,3 +53,36 @@ export async function setOAuthSession(
 ) {
   await kv.set([OAUTH_SESSIONS_PREFIX, id], value, options);
 }
+
+/**
+ * The site session is created on the server. It is stored in the database to
+ * later validate that a session was created on the server. It has no purpose
+ * beyond that. Hence, the value of the site session entry is arbitrary.
+ */
+type SiteSession = true;
+
+const SITE_SESSION_PREFIX = "site_sessions";
+
+export async function isSiteSession(id: string): Promise<boolean> {
+  const res = await kv.get<SiteSession>([SITE_SESSION_PREFIX, id]);
+  return res.value !== null;
+}
+
+export async function setSiteSession(id: string, expireIn?: number) {
+  await kv.set([SITE_SESSION_PREFIX, id], true, { expireIn });
```

Comment:
In my opinion, data like this looks redundant to store in database. The user of deno_kv_oauth should store some more meaningful data at some another key like `another_site_session_prefix, session_id` with some expireIn option. That data would be the actual 'session' and that's more appropriate for being used for sign-in checker.

@marvinhagemeister What do you think?

Reply:
Ok. Let's do this for now. (Alternative could be letting the users do site session db checks on their own, but this design also works.)

Reply:
Yeah, I'd rather this behaviour be in-built.